src/frontend/components/SøknadsSteg/2-VelgBarn/LeggTilBarn/NyttBarnKort.tsx

```diff
@@ -0,0 +1,225 @@
+                    {
+                        ident: ident.verdi,
+                        borMedSøker: borMedSøker.verdi === ESvar.JA,
+                        fødselsdato: fødselsdato.verdi,
+                        navn:
+                            navn.verdi ||
+                            intl.formatMessage({ id: 'leggtilbarn.navn-ubestemt.plassholder' }),
+                    },
+                ]),
+            },
+        });
+        nullstillSkjema();
+        settModalÅpen(false);
+    };
+
+    useEffect(() => {
+        navn.validerOgSettFelt(navn.verdi);
+    }, [navnUbestemt.verdi]);
+
+    return (
+        <>
+            <StyledModal
+                isOpen={modalÅpen}
+                contentLabel={intl.formatMessage({ id: 'leggtilbarn.popup.label' })}
+                onRequestClose={() => {
+                    settModalÅpen(!modalÅpen);
+                    nullstillSkjema();
+                }}
+            >
+                <StyledSystemtittel>
+                    <SpråkTekst id={'leggtilbarn.tittel'} />
+                </StyledSystemtittel>
+                <SkjemaGruppe>
+                    <JaNeiSpm
+                        skjema={skjema}
+                        felt={skjema.felter.erFødt}
+                        spørsmålTekstId={'leggtilbarn.er-barnet-født'}
+                    />
+                    {skjema.felter.erFødt.verdi === ESvar.NEI && (
+                        <AlertStripe type={'advarsel'} form={'inline'}>
+                            <SpråkTekst id={'leggtilbarn.feil.må-være-født'} />
+                        </AlertStripe>
+                    )}
+                </SkjemaGruppe>
+
+                <SkjemaGruppe>
+                    {skjema.felter.navn.erSynlig && (
+                        <Input
+                            {...skjema.felter.navn.hentNavInputProps(skjema.visFeilmeldinger)}
+                            label={<SpråkTekst id={'leggtilbarn.barnets-navn'} />}
+                            disabled={skjema.felter.navnUbestemt.verdi === ESvar.JA}
+                        />
+                    )}
+
+                    {skjema.felter.navnUbestemt.erSynlig && (
+                        <Checkbox
+                            {...skjema.felter.navnUbestemt.hentNavInputProps(
+                                skjema.visFeilmeldinger
+                            )}
+                            label={<SpråkTekst id={'leggtilbarn.navn-ubestemt.label'} />}
+                            onChange={event => {
+                                const { onChange } = skjema.felter.navnUbestemt.hentNavInputProps(
+                                    false
+                                );
+                                onChange(event.target.checked ? ESvar.JA : ESvar.NEI);
+                            }}
+                        />
+                    )}
+                </SkjemaGruppe>
+
+                <SkjemaGruppe>
+                    {skjema.felter.ident.erSynlig && (
```

Comment:
Ser ut som jeg ikke får opp alderen på barnet når jeg legger til testbrukeren med fødselsnummer: 27061576474
![image](https://user-images.githubusercontent.com/11887409/111965372-f495a300-8af5-11eb-8af0-3b8d26e27531.png)

Reply:
Lagt inn en rudimentær fnr => dato-funksjon i submit for å fikse dette frem til vi evt får inn en datovelger.